rfcs/select-component.md

```diff
@@ -177,10 +177,10 @@ export default () => {
   * `SelectComponentIcon: ?React.ComponentType` component for all icons appearing in Select component. It's provided `$type: ICON.loop | ICON.clearTag | ICON.clearAll | ICON.selected` to setup corresponding icon of Select component
   * `Tag: ?React.ComponentType` component for selected options Tags shown in Input for multiple mode selection
   * `SelectSpinner: ?React.ComponentType` component for dropdown spinner if async options are loading
-* `onChange: func(e: SyntheticInputEvent, params: Object)`:
-  handler for events on trigger element when option are changing selection.  `params` has `type` indicating which action is performed, current `Option` with `id` and `label` of selected\unselected option and `selectedOptions` array of all of selected.
-* `onTextInputChange: func(e: SyntheticInputEvent)`:
-  handler for events on trigger element when text of search input (in Search mode has changed).
+* `onSelect: func({event: SyntheticInputEvent, type, option, selectedOptions})`:
+  handler for events on trigger element when option are changing selection.  The `type` indicating which action is performed, current `Option` with `id` and `label` of selected\unselected option and `selectedOptions` array of all of selected.
+* `onChange: func`:
+  handler for events on trigger element
```

Comment:
It may not be clear what is a `trigger element`, should we say something like`onChange handler for an input element`?